src/graphql-server/schema.graphql

```diff
@@ -106,6 +107,23 @@ type HposAuth {
   isAuthed: Boolean
 }
 
+input HolofuelUserInput {
+  id: ID
+  nickname: String
+}
+
+input CancelDeclinedTransactionsInput {
+  id: ID # ID will be the origin commit hash
+  amount: Float
+  counterparty: HolofuelUserInput
+  direction: TransactionDirection
+  status: TransactionStatus
+  type: TransactionType
+  timestamp: String
+  fees: Float
+  notes: String
+}
+
```

Comment:
I don't think we need this Type.

See comment on `holofuelRefundAllDeclined` below

Reply:
So, unfortunately, I learned the hard way we needed this. After some research, I found this solution read this one straight from the Apollo docs. - Exchanging this for a list of inline args does not though, as not all of the args are primitive types.

That said, please let me know if you have alternative knowledge or insights.

Reply:
We only need to pass in the ids. See comment on `holofuelRefundAllDeclined` below.